packages/bundle-size/package.json

```diff
@@ -1,12 +1,24 @@
 {
   "name": "@fluentui/bundle-size",
   "version": "1.0.0",
-  "private": true,
   "bin": {
     "bundle-size": "./bin/bundle-size.js"
   },
   "scripts": {
     "lint": "just-scripts lint",
     "type-check": "tsc"
+  },
+  "dependencies": {
+    "@babel/core": "^7.10.4",
+    "glob": "^7.1.2",
```

Comment:
All these deps are production they probably should carets...

Reply:
yeah this is gonna make syncpack cry...